test/compress/template-string.js

```diff
@@ -309,9 +309,9 @@ keep_raw_content_in_tagged_template: {
         evaluate: true
     }
     input: {
-        var foo = tag`\u0020\u{20}\u{00020}\x20\40\040 `;
+        var foo = tag`\u0020\u{20}\u{00020}\x20 `;
```

Comment:
Why was the octal escape removed?

The ES2018 template revision allows for invalid octal escapes within tagged template literals. This PR should not error out in such a case:

```
$ echo 'console.log(String.raw`\u0020\u{20}\u{00020}\x20\40\040`);' | node-v10.4.1
\u0020\u{20}\u{00020}\x20\40\040
```
```
$ echo 'console.log(String.raw`\u0020\u{20}\u{00020}\x20\40\040`);' | terser
Parse error at 0:1,22
console.log(String.raw`\u0020\u{20}\u{00020}\x20\40\040`);
                      ^
ERROR: Octal escape sequences are not allowed in template strings
```